client/src/components/Footer/Footer.jsx

```diff
@@ -5,13 +5,13 @@ import("./Footer.css");
 export default function Footer() {
   return (
     <div className="footer-container">
-      <NavLink to="#" className="footer-link">
+      <NavLink to="/about" className="footer-link">
         À propos
       </NavLink>
       <NavLink to="#" className="footer-link">
         Nous contacter
       </NavLink>
-      <NavLink to="#" className="footer-link">
+      <NavLink to="/conditionsgénérales" className="footer-link">
```

Comment:
never use special characters in your urls /conditionsgénérales -> /conditionsgenerales

Reply:
Done